app/assets/stylesheets/building_map.scss

```diff
@@ -53,6 +53,52 @@
   text-align: center;
 }
 
+.pin-icon1 {
+  width: 20px;
+  height: 20px;
+  border-radius: 50% 50% 50% 0;
+  background: #CA091A;
+  position: absolute;
+  transform: rotate(-45deg)  translate(9px, -20px);
+  left: 50%;
+  top: 50%;
+}
+
+.pin-icon1::before {
+  content: '1';
+  width: 14px;
+  height: 14px;
+  margin: 3px 0 0 3px;
+  background: #fff;
+  position: absolute;
+  border-radius: 50%;
+  transform: rotate(45deg);
+  text-align: center;
+}
+
+.pin-icon2 {
+  width: 20px;
+  height: 20px;
+  border-radius: 50% 50% 50% 0;
+  background: #CA091A;
+  position: absolute;
+  transform: rotate(-45deg)  translate(9px, -20px);
+  left: 50%;
+  top: 50%;
+}
+
+.pin-icon2::before {
+  content: '2';
+  width: 14px;
+  height: 14px;
+  margin: 3px 0 0 3px;
+  background: #fff;
+  position: absolute;
+  border-radius: 50%;
+  transform: rotate(45deg);
+  text-align: center;
+}
+
```

Comment:
This also smells like code duplication because it's similar to `.pin-icon1::before`